Support non-fail-fast execution of test cases (#79)
apache/skywalking-infra-e2e: docs/en/setup/Configuration-File.md

  retry:            # verify with retry strategy
    count: 10       # max retry count
    interval: 10s   # the interval between two attempts, e.g. 10s, 1m.

>   fail-fast: false  # when a case fails, whether to stop verifying other cases. This property defaults to false.

Reviewer: What about setting the default value to `true` so it behaves just like before?

Author: OK! Delayed a little time because of lunch. I will fix as soon as possible!

Reviewer: > OK! Delayed a little time because of lunch. I will fix as soon as possible!

Please don't be stressful, take your time 🍖

Author: 😊

Reviewer: Looks like you just modify the value in example config file and doc, but in codes the default value is still `false`. It seems there is no easy way to set a default value for `bool` in struct. @fgksgf any idea?

Reviewer 2: How about setting the default `true` here https://github.com/apache/skywalking-infra-e2e/blob/82494d053f879c2e10bc6c1439c05673d50bb37b/internal/config/globalConfig.go#L41

Reviewer: @chunriyeqiongsaigao please read this 👆

Author: OK, I will write the init() function!

        - path/to/cases.yaml            # cases file path
```

> The test cases are executed in the order of declaration from top to bottom.When verifying a case, if the execution fails, and the retry strategy is exceeded, whether to stop verifying other cases depends on the value of "fail-fast". If the value of "fail-fast" is "false", the process will continue to verify the other cases.

Reviewer 2: ```suggestion
The test cases are executed in the order of declaration from top to bottom. When the execution of a case fails and the retry strategy is exceeded, it will stop verifying other cases if `fail-fast` is `true`. Otherwise,  the process will continue to verify other cases.
```
